scripts/mutate_dataset.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
```

Comment:
Was this file just for your testing? Or is it meant to be committed to the repo? If so, maybe we could add a bit of documentation on how a dev should use it?

Reply:
I'll add a docstring, it was for taking a snippet of a snapshot and randomly altering some data to use in test fixtures. to simulate the same people appearing with changes over time. I used it two create the 2011 dataset added in this PR.

Reply:
:+1: